### PR TITLE
Create express app

### PR DESCRIPTION
**Pull request by Ripraskel**
Created pull request in order to review.

### CODE REVIEW

File: express-app/config.js
Comment (Ripraskel) [inline express-app/config.js:1]: not good practice to commit empty files.

File: express-app/index.js
Comment (Ripraskel) [inline express-app/index.js:1]: Have you tested this endpoint by using postman?

File: express-app/package.json
Comment (Ripraskel) [inline express-app/package.json:1]: No start script in this package.json

File: express-app/services/messagingService.js
Comment (Ripraskel) [inline express-app/services/messagingService.js:1]: What is this for?

File: node_modules/.bin/mime
Comment (Ripraskel) [inline node_modules/.bin/mime:1]: node_modules should never be committed to source control.

File: package.json
Comment (Ripraskel) [inline package.json:1]: Was this your intent to create a package.json a level above the apps? why?

Comment (fadyawan) [reply package.json:1]: Accident

File: express-app/index.js
Comment (fadyawan) [reply express-app/index.js:1]: yes

**Pull request merged**